tests/integration_tests/functional/test_aarch64_armshaker.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Opcode fuzzer test for aarch64."""
+
+# pylint: disable=too-many-lines
+
+import platform
+import time
+
+import pytest
+
+from framework import utils
+from framework.utils_cpu_templates import nonci_on_arm
+
+PLATFORM = platform.machine()
+
+TIMEOUT = 4 * 60 * 60
+
+
+def test_armshaker(vm):
```

Comment:
Is this a test or a support function? If the latter (it looks like by the usage), it should have a different name, like `run_armshaker` because pytest collects all `test_*` functions as tests.

Reply:
fixed